pydoro/pydoro_core/config.py

```diff
@@ -95,10 +95,16 @@ def _ini_parse(self):
     def _create_default_ini(self, filename):
         """
         Creates default ini configuration file
-        Saves it in '~/.pydoro.ini' or the location specified by PYDORO_CONFIG_FILE environment variable
+        Saves it in '~/.config/pydoro/pydoro.ini' or the location specified by PYDORO_CONFIG_FILE environment variable
         """
-        with open(filename, "w+") as configfile:
-            self._conf.write(configfile)
+        try:
+            with open(filename, "w+") as configfile:
+                self._conf.write(configfile)
+                
+        except FileNotFoundError:
+            config_dir = os.path.expanduser("~/.config/pydoro")
+            os.makedirs(config_dir, exist_ok=True)
```

Comment:
Can we move this before attempting to write the file?

Reply:
Done :)